test/fixtures/prx_story_all.json

```diff
@@ -1,4 +1,75 @@
 {
+  "appVersion": "v4",
```

Comment:
I rearranged this to make it easier for me to look through, but the real change is the addition of the 4 `prx:storage` links, one to each audio file